### DISCUSSION TITLE
IBAX functionality

### DISCUSSION THREAD

**Discussion by yusufeweqewew** [IBAX-io/go-ibax#730] [Soru Cevap - Türkçe]
When will IBAX be fully functional ?

**Comment by Yusufknk** [comment]
Airdroptan sonra Coinmarket ve Coingecko'da listelenecektir. Ardından ön satış işlemleri olacaktır. Gelişmeleri sosyal medya kanallarında takip edebilirsiniz.

**Reply by yusufeweqewew** [reply]
teşekkürler

**Reply by Garadepelim** [reply]
Teşekkürler
